### DISCUSSION TITLE
Is there a way to always flatten beams?

### DISCUSSION THREAD

**Discussion by mtoneil** [0xfe/vexflow#1438] [Q&A] [answered]
Vexflow sometimes renders beams at an angle like this:

![image](https://user-images.githubusercontent.com/1349844/190661164-4287b4c7-e6cf-494d-9bdf-49ed3774692e.png)

Is there a way to force the beams to flatten, so the beams in beats 3 and 4 in the above example would look more like beats 1 and 2?
(Note: I am not using auto-beaming.)

**Comment by sschmidTU** [comment, marked as answer]
`beam.render_options.flat_beams = true`

Also have a look at `flat_beam_offset` and `min_flat_beam_offset` (in `render_options`)
